R/conversion.R

```diff
@@ -43,6 +43,17 @@ convert <- function(value, from, to) {
   x
 }
 
+`units<-.logical` <- function(x, value) {
+  if(is.na(x)) {
+    c <- match.call()
+    c[1] <- call('units<-.numeric')
+    c[['x']] <- as.numeric(x)
+    eval(c)
```

Comment:
I'd prefer to use .NextMethod() here, as in
```
x <- as.numeric(x)
NextMethod()
```

Reply:
NextMethod() does not appear to work here because the logical values provided only have a single type and not a hierarchy. There may be a way to make it work but it could end up more complex than my version. Interested to see any other ways to do this.

Reply:
Thanks for trying, let's leave it like this.